diff --git a/component.mk b/component.mk
index ce409c7..8cd9c66 100644
--- a/component.mk
+++ b/component.mk
@@ -36,4 +36,4 @@ $(wifi_config_OBJ_DIR)/src/wifi_config.o: $(wifi_config_OBJ_DIR)/content/index.h
 $(wifi_config_OBJ_DIR)/content/index.html.h: $(WIFI_CONFIG_INDEX_HTML)
 	$(vecho "Embed %<")
 	$(Q) mkdir -p $(@D)
-	$(Q) $(wifi_config_ROOT)/tools/embed.py $< > $@
+	$(Q) python $(wifi_config_ROOT)/tools/embed.py $< > $@
